tensorflow/core/framework/device.h

```diff
@@ -178,6 +178,8 @@ class Device : public DeviceBase {
 
   virtual bool IsLocal() const { return true; }
 
+  virtual bool IsRemoteCallAllowed() const;
```

Comment:
Could you please add a comment for this method? Thanks!

Reply:
Sorry, missed that when resolving merge. Done.

Reply:
Sorry, missed that when resolving merge conflict. Done.